docs/adr/0019-keep-study-as-a-dto.md

```diff
@@ -0,0 +1,32 @@
+# Keep study as a DTO
+
+## Context and Problem Statement
+
+The study holds query and library entries that could be replaced respectively with complex query and fetcher instances.
+This poses the question: should the study remain a pure DTO object or should it contain direct object instances?
+
+## Considered Options
+
+* Keep study as DTO and use transformers
+* Replace entries with instances
+
+## Decision Outcome
+
+Chosen option: "Keep study as DTO and use transformators", because comes out best (see below).
+
+## Pros and Cons of the Options
+
+### Keep study as DTO and use transformators
+
+* Good, because makes GUI implementation probably easier down the line
```

Comment:
For the GUI, you probably need a wrapper around the study class anyway (and then it doesn't matter where the data is coming from).

Reply:
> So we cannot use the jackson library? What about the other ones? Is there a well-maintend one that is compatible with the module system?

Is this even related to jackson specifically? To me the amount of added modules seems to be problematic?

Reply:
My guess is that jackson has a long module list which gives this error. But it could also be that all of our dependencies together are already at the borderline, and adding jackson just tipped us over the limit - this would be really bad indeed.

Should be easy to test though...

Reply:
Yeah, ill proceed with some testing now...